app/components/candidate_interface/application_dashboard_guidance_component.html.erb

```diff
@@ -0,0 +1,32 @@
+<% if no_offers_awaiting_decisions? %>
+
+  <% if awaiting_multiple_decisions? %>
+    <p class="govuk-body">Your applications have been submitted and are with the training providers.</p>
```

Comment:
super nitty and probably worth ignoring but it could be two applications choices with the same Provider